weblate_web/templates/snippets/reviews.html

```diff
@@ -0,0 +1,384 @@
+{% load i18n %}
+{% load static %}
+
+<style>
+
+.testimonial1{
+    width: 365px;
+    height: 347px;
+    border-radius: 30px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 10px;
+}
+
+.testimonial2{
+    height: 291px;
+    width: 365px;
+    border-radius: 30px;
+    background-color: #FFFFFF;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 33px;
+}
+
+.testimonial3{
+    width: 365px;
+    height: 347px;
+    border-radius: 30px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 10px 0 rgba(0,0,0,0.1);
+    margin-top: 10px;
+}
+
+.icon-review{
+    text-align: left;   
+}
+.icon-review img{
+    margin-left: 50px;
+}
+.icon-review p{
+    margin-left:35px;
+    margin-top: -25px;
+    width: 70%;
+    font-size: 16px;
+    line-height: 20px;
+    letter-spacing: 0.5px;
+    color:#2A3744 ;
+}
+
+.circle-user{
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    position: relative;
+    left: 32px;
+    margin-top: 20px;
+}
+.circle-user img{
+    margin: 0px;
+    border-radius: 50%;
+}
+
+.icon-user{
+    display: flex;
+    text-align: left;
+}
+.contentt{
+    margin-top: 20px;
+    margin-left: 45px;
+}
+.contentt a{
+    color: #1FA385;
+    line-height: 23px;
+    text-decoration: underline;
+}
+.unqimg{
+    margin: 0px !important;
+}
+.custom-btn {
+    margin-top: 20px;
+}
+.custom-btn a{
+  text-decoration: none;
+}
+
+</style>
+
+<style>
```

Comment:
This style block looks like not intended changes. It resets existing page styling.

Reply:
I have gotten rid of the excessive CSS.